Tests/service-test.yml

```diff
@@ -0,0 +1,65 @@
+config:
+  target: "http://localhost:4006"
+  http:
+    timeout: 10
+  phases:
+    - duration: 60
+      arrivalRate: 10
+      rampTo: 50
+    - duration: 60
+      arrivalRate: 50
+      rampTo: 75
+    - duration: 60
+      arrivalRate: 75
+      rampTo: 100
+    - duration: 300
```

Comment:
Is there a reason your duration here is 300 and not 60 like the others? Also is there a reason you're not trying 1000 RPS? Maybe 1000 was too much for you system so you downgraded to go up to 200?

Reply:
For the duration question, I wanted to slowly ramp up from 100 to 200 to better see if it was the sustained load or actual stress that was causing the high cpu usage. I could have continued to increase smaller amounts in 60 second increments, but slowly increasing over 5 minutes seemed like less code for the same effect.

For the requests question, each scenario actually launches at least 4 requests, so 200 users doing one page view is actually 800 requests. I also have the scenario for one user to do five page views over a period of time, so all in all they'd launch 20 requests each. It worked out to where I hit a peak of nearly 1600 rps, which came back down to a consistent 1300 rps after I bounced off that ceiling.